docs/legacy/getting-started-apm-server.asciidoc

```diff
@@ -1,34 +1,156 @@
+// === Standalone Elastic Agent-managed APM Server
+// // I really don't know how to sell this option
+// Instead of installing and configuring the APM Server binary, let {agent} orchestrate it for you.
+// Install {agent} and manually configure the agent locally on the system where it's installed.
+// You are responsible for managing and upgrading {agent}. This approach is recommended for advanced users only.
+
+// **Pros**:
 
-Skip managing your own {es}, {kib}, and APM Server by using our
-{ess-product}[hosted {es} Service] on
-{ecloud}.
+// - Easily add integrations for other data sources
+// useful if EA already in place for other integrations, and customers want to customize setup rather than using Fleet for configuration
+// // TODO:
+// // maybe get some more hints on this one from the EA team to align with highlighting the same pros & cons.
 
-image::images/apm-architecture-cloud.png[Install Elastic APM on cloud]
+// **Available on Elastic Cloud**: ❌
 
-{ess-trial}[Try out the {es} Service for free],
-then see {cloud}/ec-manage-apm-settings.html[Add APM user settings] for information on how to configure Elastic APM.
+// This supports all of the same outputs as binary
+// see https://github.com/elastic/apm-server/issues/10467
+// **Supported outputs**:
+
+// **Configuration method**: YAML
+
+// image::./images/ea-ov.png[APM Server ea overview]
+
+// @simitt's notes for how to include EA-managed in the decision tree:
+// ****
+// If we generally describe Standalone Elastic Agent managed APM Server then we should also add it to this diagram:
+// Do you want to use other integrations?
+// -> yes: Would you like to use the comfort of Fleet UI based management? -> yes: Fleet managed APM Server; -> no: Standalone Elastic Agent managed APM Server
+// -> no: What is your prefered way of configuration? -> yaml: APM Server binary; -> Kibana UI: Fleet managed APM Server
+// ****
+
+// Components required:
+
+// [options="header"]
+// |====
+// | Installation method                         | APM Server | Elastic Agent | Fleet Server
+// | APM Server binary                           | ✔️          |               |
+// // | Standalone Elastic Agent-managed APM Server | ✔️          | ✔️             |
+// | Fleet-managed APM Server                    | ✔️          | ✔️             | ✔️
+// |====
 
 [float]
-=== Install and manage the stack yourself
+=== Help me decide
+
+Use the decision tree below to help determine which method of configuring and running the APM Server is best for your use case.
+
+[subs=attributes+]
+include::../diagrams/apm-decision-tree.asciidoc[APM Server decision tree]
 
-If you'd rather install the stack yourself, first see the https://www.elastic.co/support/matrix[Elastic Support Matrix] for information about supported operating systems and product compatibility.
+
+=== APM Server binary
+
+// *******************************************************
+// STEP 0
+// *******************************************************
+
```

Comment:
This now shows up as an empty page, should we just remove the _Prerequisites_ header?